### PR TITLE
Disabled 'Unable to Determine and Decline to State' options in otherEthnicity

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**

### CODE REVIEW

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 1]
```diff
@@ -12,6 +12,15 @@ import {
 
 import { formatTable } from '../../../_utils/formatters';
 
+const OTHER_ETHNICITY = [
+  {
+    value: 'Unable to determine*',
```
Comment (kelfink) [inline app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: Who determined this value ?  Is it in the business rule?

Comment (Sushmareddy-OSI) [reply app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: one of the options for otherEthnicity field (dropdown)

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 2]
```diff
@@ -12,6 +12,15 @@ import {
 
 import { formatTable } from '../../../_utils/formatters';
 
+const OTHER_ETHNICITY = [
```
[the remark this answers is not in the document]
Comment (Sushmareddy-OSI) [reply app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: other_ethnicity is a name of the field so I used same name for constants(options) which will be relevant with the field name

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 3]
```diff
@@ -12,6 +12,15 @@ import {
...
+  {
+    value: 'Unable to determine*',
+    label: 'Unable to determine',
+    disabled: true,
+  },
+  { value: 'Declined to state*', label: 'Declined to state', disabled: true },
```
[the remark this answers is not in the document]
Comment (Sushmareddy-OSI) [reply app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: 'Decline to state*' is the option name that we are receiving from API

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js (hunk 1 above)
Comment (kelfink) [reply app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: Value is the same as it is in the legacy system (per discussion)

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (kelfink) [reply app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: Yeah, it does at least have the same name.  Ok.

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 4]
```diff
@@ -163,7 +172,7 @@ export default class RaceAndEthnicity extends React.Component {
               id="dropdown5"
               gridClassName="col-md-12 col-sm-6 col-xs-12"
               selectedOption={this.state.childClient.other_ethicity}
-              options={RACE}
+              options={OTHER_ETHNICITY}
               label="Other Ethnicity"
               onChange={this.handleOtherEthnicity('other_ethicity')}
```
[the remark this answers is not in the document]
Comment (kelfink) [reply app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: Ed and I reviewed the existing code and it looks like line 168 here needs to change to 'other_ethnicity' while line 165 also needs this change.

**Pull request merged**